meta/mm_sequence.py

```diff
@@ -175,6 +203,20 @@ def update_rqmt(rqmt, extra):
             else:
                 raise ValueError("Unknown action: %s" % action)
 
+    def get_alignment_score_report(self):
+        """
+        :return: report .txt file path containing the alignment scores in order of the job sequence
+        """
+        if self.report_job is not None:
+            return self.report_job.out_report
+
+        logs = []
+        for job, log in zip(self.all_jobs, self.all_logs):
+            if isinstance(job, AlignmentJob):
+                logs.append(log)
+        self.report_job = AMScoresFromAlignmentLogJob(logs)
```

Comment:
```suggestion
        if self.report_job is None:
            logs = []
            for job, log in zip(self.all_jobs, self.all_logs):
                if isinstance(job, AlignmentJob):
                    logs.append(log)
            self.report_job = AMScoresFromAlignmentLogJob(logs)
```

just a suggestion that I think is simpler but I am fine either way.